src/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --acdc-color: #00acdc;
+  --white-color: #fff;
+
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
+  }
+
+  &__product-name {
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--main-text-color);
+
+    margin: 0;
```

Comment:
I'd recommend you to reset default margin for `p` and `h2` tags to avoid resetting this value each time